gateleen-hook/src/test/java/org/swisspush/gateleen/hook/HookHandlerTest.java

```diff
@@ -479,11 +489,11 @@ public void listenerRegistration_rejectNotWhitelistedHttpMethods(TestContext tes
             }
 
             // Trigger
-            hookHandler.handle( request );
+            hookHandler.handle(request);
 
             { // Assert request got rejected.
-                testContext.assertTrue( statusCodePtr[0] >= 400 );
-                testContext.assertTrue( statusCodePtr[0] <= 499 );
+                testContext.assertTrue(statusCodePtr[0] >= 400);
```

Comment:
Why not checking that status code is 400 when requests with these methods are made?

Reply:
Hmm. Yes would be an option of course. Didn't think about that. Will do 👍